content/manager/delete-blueprint.md

```diff
@@ -8,11 +8,18 @@ weight: 800
 
 Deleting a blueprint removes its model from the database and deletes its resources from the fileserver. Deleting a blueprint does not delete the deployments created from that blueprint or resources of those deployments.
 
-To delete a blueprint from the CLI, run
+To delete a blueprint run the following command.
 
 {{< gsHighlight  bash >}}
 cfy blueprints delete [OPTIONS] BLUEPRINT_ID
 {{< /gsHighlight >}}
 
-To delight a blueprint from the Cloudify Web interface, on the Blueprints widget, click **Delete** on the relevant blueprint.
+To delete a blueprint from the Cloudify Web interface, on the Blueprints widget, click **Delete** on the relevant blueprint.
```

Comment:
To delete a blueprint from the Cloudify Web interface, go to the Blueprints widget and click **Delete** on the relevant blueprint.